server/lib/parties/party-service.test.ts

```diff
@@ -140,8 +141,7 @@ describe('parties/party-service', () => {
     webClient = connector.connectClient(webUser, WEB_USER_CLIENT_ID, 'web')
 
     clearTestLogs(nydus)
-    findUsersByNameMock.mockClear()
-    findUsersByIdAsMapMock.mockClear()
+    jest.clearAllMocks()
```

Comment:
This is not behaviorally the same as what was happening previously, if you need to fix something specific then please just add in the mocks you want to clear. Blindly clearing every mock in the system makes it impossible to compose different test utilities safely

Reply:
Yeah, I did try to at least make sure I didn't break anything with this change :d

I thought that clearing all mocks before each test would be a good testing practice to adopt, but I'm not sure where I read that now ^^. Reverted the change for now.

Reply:
I don't think this is as bad as I thought it was (I thought this was doing the thing that `mockReset` turns out to do, that is, clearing mocked implementation/values as well), but I would still avoid doing this at a blanket level. It's not possible to opt out of this should you need to, other than by going through and finding every mock in the test that you need to reset. It's much better to be prescriptive when you're writing the test, "I know specifically this mock has calls I want to ignore/drop at this time". `clearAllMocks` affects every mock everywhere, so if we add e.g. a test utility that maintains its own mock and provides assertions against that mock, you might be screwing up its functionality and it will absolutely not be obvious when those tests fail where the cause lies.